pkg/pillar/cmd/upgradeconverter/persistlayout.go

```diff
@@ -22,7 +22,7 @@ import (
 func convertPersistVolumes(ctxPtr *ucContext) error {
 	log.Functionf("convertPersistVolumes()")
 	checkpointFile := ctxPtr.configCheckpointFile()
-	if !fileExists(checkpointFile) {
```

Comment:
Note that this fileExists checks os.IsNotExist. It probably makes sense to add that check to fileutils.FileExists (I don't think it has such a check and just check that os.Stat succeeds.)